<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OprNetConfig</class>
 <widget class="QDialog" name="OprNetConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>611</width>
     <height>351</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QWidget" name="">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>set address operator(string)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="addressOpr"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>set port operator(int)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="portOpr"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>set address comander(string)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="addressCmd"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>set port comander(int)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="portCmd"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
